plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -289,6 +289,7 @@ public PostgreSqlClient(
                 .map("$not($is_null(value))").to("value IS NOT NULL")
                 .map("$not(value: boolean)").to("NOT value")
                 .map("$is_null(value)").to("value IS NULL")
+                .map("$nullif(first, second)").to("NULLIF(first, second)")
```

Comment:
When copying this to eg MySQL we need to remember to drop this for varchar (case-sensitivity of the implicit comparison).